cg/meta/workflow/analysis.py

```diff
@@ -401,27 +407,18 @@ def decompress_case(self, case_id: str, dry_run: bool) -> None:
             self.set_statusdb_action(case_id=case_id, action="analyze")
         LOG.info("Decompression started for %s", case_id)
 
-    def resolve_decompression(self, case_id: str, dry_run: bool) -> bool:
+    def resolve_decompression(self, case_id: str, dry_run: bool) -> None:
         """
-        Handles decompression automatically for case.
-        Return True if decompression is running
+        Decompresses a case if needed and adds new fastq files to Housekeeper.
         """
         if self.prepare_fastq_api.is_spring_decompression_needed(case_id):
             self.decompress_case(case_id=case_id, dry_run=dry_run)
-            return True
+            return
 
         if self.prepare_fastq_api.is_spring_decompression_running(case_id):
-            self.set_statusdb_action(case_id=case_id, action="analyze")
```

Comment:
This is a change in behaviour. I'm in favour of it but we should consult production

Reply:
Spoke with @karlnyr about this and he quite like the functionality. We agreed though that it could be moved into the actual start command to make the code less complex

Reply:
@Vince-janv So do we revert this change for this PR and create a later PR where we move it?

Reply:
I think that's the best plan of action, as not to change too much at once 👍